as/thrift.js

```diff
@@ -47,8 +47,15 @@ function TChannelAsThrift(opts) {
     self.thriftSource = opts.source;
     self.thriftFileName = opts.thriftFileName || 'service.thrift';
     self.spec = new thriftrw.Thrift({
+        entryPoint: opts.entryPoint,
+        idls: opts.idls,
+        thrifts: opts.thrifts,
         source: self.thriftSource,
-        strict: opts.strict
+        strict: opts.strict,
+        allowFilesystemAccess: opts.allowFilesystemAccess,
+        allowIncludeAlias: opts.allowIncludeAlias,
+        fs: opts.fs,
+        memo: opts.memo
```

Comment:
Let’s not expose the memo at this time. It would be harder to remove than to add when we need it.